helm-charts/sep-service/templates/configmap.yaml

```diff
@@ -263,3 +261,17 @@ data:
     ORG_LOGO = "https://myorglogo.png"
     ORG_SUPPORT_EMAIL="myname@myemail.org"
     {{- end }}
+---
```

Comment:
I think we should make this entire map optional. 
ie perhaps move {{- if .Values.assets }} to top (above the + ---)?? 
the reason is that bitso provides their config map via a custom volume mount.  If we ship it like this, we could break their implementation until they move to  providing via values file.

Reply:
good call, I'll make that change